UnitedIncome/components
# [APP-5034] Add Folder Circle Illustrations

### Description
This adds the folder circle illustrations provided by April to the component library and Storybook. This will be used in the new personal information McG flow.

### Testing Instructions
1. Launch repo locally
2. Observe the new illustrations

### Screenshots
[![Image from Gyazo](https://i.gyazo.com/06c202fe1e395286005e4e3b80fdaf91.gif)](https://gyazo.com/06c202fe1e395286005e4e3b80fdaf91)





### Comments
I ran the SVGs through ImageOptim :) 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5034)
